### ISSUE TITLE
Non discript error when loading resource

### ISSUE THREAD

**Issue by doodirock** [RESTful-Drupal/restful#765]
First off, LOVE the module.  

I've recently implemented some custom resources, but I'm getting back an error on my base api and all resource that I can't seem to figure out.  It returns back a broken JSON document containing the following message.

`{"type":"http:\/\/www.w3.org\/Protocols\/rfc2616\/rfc2616-sec10.html#sec10.4.5","title":"Invalid URL path.","status":404,"detail":"Not Foun`

As you can see it says the URL is invalid even if I just go to the base API.  It also appears to be broken as it just ends after detail with "Not Foun" and no closing bracket.  If it helps I'm currently working on Pantheon.

I'm having trouble attempting to debug.

**Comment by doodirock** [comment]
Looks like its a conflict with RestWS.  I was hoping to build a new API while keeping RestWS enabled since its what I'm currently using in production.  I suppose I'll just have to build this off to the side in order to test on

**Comment by e0ipso** [comment]
That's weird. There should not be an interaction between RestWS and RESTful.

Re-opening.

**Comment by e0ipso** [comment]
Also I'm assuming this is for `2.x`.

**Comment by doodirock** [comment]
Yep its for 2.x.  Not sure why, but when ever i turn it on I get the error.  I rebuilt my registry and cleared cache just in case.  I'll do some more testing tomorrow and see what I can find.

**Comment by e0ipso** [comment]
@doodirock can you provide a set of steps to reproduce from a clean installation?
